### ISSUE TITLE
getInboundReferences fails on json parsing

### ISSUE THREAD

**Issue by polina-c** [dart-lang/sdk#51100]
Versions:
 - devtools: https://github.com/polina-c/devtools/tree/99a59cf471f5f0633947970f05f71aad2383b17c
 - flutter: 
 ```
Flutter 3.7.0-15.0.pre • channel flutter-3.7-candidate.15 • https://github.com/flutter/flutter.git
Framework • revision a02b9d2bff (2 weeks ago) • 2023-01-09 10:29:26 -0800
Engine • revision 51baed6e01
Tools • Dart 3.0.0 (build 3.0.0-97.0.dev) • DevTools 2.20.0
```
Repro:

1. Start devtools with experiments enabled
1. On inspector page select a widget
2. Expand references>live>inbound
3. Find error in console

![Screenshot 2023-01-23 at 7 10 30 AM](https://user-images.githubusercontent.com/12115586/214036636-253bda73-9d6f-4b97-aa82-1e0d23786333.png)

![Screenshot 2023-01-23 at 7 00 53 AM](https://user-images.githubusercontent.com/12115586/214034999-e3454cd0-0123-46fc-a9c1-05ae96e9e291.png)

**Comment by polina-c** [comment]
Fixed in v 10 of VM Service

**Comment by a-siva** [comment]
//cc @derekxu16 @bkonyi please close the issue if it is already fixed.

**Comment by polina-c** [comment]
It is still here in v10:

<img width="730" alt="Screenshot 2023-02-07 at 10 28 48 AM" src="https://user-images.githubusercontent.com/12115586/217334087-c62ddbda-1588-4a69-8c11-5478d1f8b51b.png">

**Comment by derekxu16** [comment]
Fixed in [`package:vm_service v11.0.1`](https://pub.dev/packages/vm_service/versions/11.0.1).
